src/ProxyClient/Symfony.php

```diff
@@ -52,10 +54,37 @@ public function refresh($url, array $headers = [])
     protected function configureOptions()
     {
         $resolver = parent::configureOptions();
-        $resolver->setDefaults([
-            'purge_method' => PurgeListener::DEFAULT_PURGE_METHOD,
-        ]);
+        $resolver->setDefault('purge_method', PurgeListener::DEFAULT_PURGE_METHOD);
+        $resolver->setAllowedTypes('purge_method', 'string');
+        $resolver->setDefault('purge_tags_method', PurgeTagsListener::DEFAULT_PURGE_TAGS_METHOD);
+        $resolver->setAllowedTypes('purge_tags_method', 'string');
+        $resolver->setDefault('purge_tags_header', PurgeTagsListener::DEFAULT_PURGE_TAGS_HEADER);
+        $resolver->setAllowedTypes('purge_tags_header', 'string');
+        $resolver->setDefault('purge_tags_header_length', 7500);
+        $resolver->setAllowedTypes('purge_tags_header_length', 'int');
```

Comment:
we should perhaps align naming with Varnish one or the other way?

Reply:
Also code wise easier to read if `setDefaults()` is used imo: https://github.com/FriendsOfSymfony/FOSHttpCache/pull/332/files#diff-006816ea733cc0564d6a5b345ed47929R160

Reply:
agreed with setDefaults. for the naming i agree as long as its really the same semantics. we can't change existing names easily, because backwards compatibility is important.